docs/en/04_Changelogs/4.4.0.md

```diff
@@ -3,12 +3,31 @@
 ## Overview {#overview}
 
  - [Correct PHP types are now returned from database queries](/developer_guides/model/sql_select#data-types)
- 
+ - The name of the directory where vendor module resources are exposed can now be configured by defining a `extra.resources-dir` key in your `composer.json` file. If the key is not set, it will automatically default to `resources`. New projects will be preset to `_resources`. This will avoid potential conflict with SiteTree URL Segments.
+ - dev/build is now non-destructive for all Enums, not just ClassNames. This means your data won't be lost if you're switching between versions, but watch out for code that breaks when it sees an unrecognised value!
+
 ## Upgrading {#upgrading}
 
- - dev/build is now non-destructive for all Enums, not just ClassNames. This means your data won't be lost if you're switching between versions, but watch out for code that breaks when it sees an unrecognised value!
+### Adopting to new `_resources` directory
+
+1. Update your `.gitignore` file to ignore the new `_resources` directory. This file is typically located in the root of your project or in the `public` folder.
+2. Add a new `extra.resources-dir` key to your composer file.
+```js
+{
+    // ...
+    "extra": {
+        // ...
+        "resources-dir": "_resources"
+    }
+}
+```
+3. Expose your vendor assets by running `composer vendor-expose`.
+4. Remove the old `resources` folder. This folder will be located in the `public` folder if you have adopted the public web root, or in the root of your project if you haven't.
+
+You may also need to update your server configuration if you have applied special conditions to the `resources` path.
 
 ## Changes to internal APIs
 
  - `PDOQuery::__construct()` now has a 2nd argument. If you have subclassed PDOQuery and overridden __construct() 
```

Comment:
```suggestion
 - `PDOQuery::__construct()` now has a 2nd argument. If you have subclassed PDOQuery and overridden `__construct()` 
```

To prevent flow on bold highlighting (see GitHub preview) - I don't think our docs site renders it this way though, just a minor thing